SharedSources/MediaLibraryModel/CollectionModel.swift

```diff
@@ -40,7 +40,13 @@ class CollectionModel: MLBaseModel {
     var indicatorName: String = "Collections"
 
     func delete(_ items: [VLCMLObject]) {
-       assertionFailure("still needs implementation")
+        if let playlist = mediaCollection as? VLCMLPlaylist {
+            for item in items where item is VLCMLMedia {
+                if let index = files.firstIndex(of: item as! VLCMLMedia) {
```

Comment:
What do you think of something like this:

``` 
for case let media as VLCMLMedia in items {
    if let index = files.firstIndex(of: media) {
        ...
    }
```

Reply:
didn't know that was possible :D will do